README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
-   or [OpenJDK](https://openjdk.java.net/install/).  Java 8 is required.  To install Java, run
-  * Mac
-      ```
-      brew install openjdk@8
-      ```
-      OpenJDK 8 may need to be added to your path:
-      ```
-      echo 'export PATH="/usr/local/opt/openjdk@8/bin:$PATH"' >> ~/.zshrc
-      ```
-  * Windows WSL2 - Ubuntu
-      ```
-      sudo apt update
-      sudo apt install openjdk-8-jdk-headless
-      ```
-  * Linux
-      ```
-      sudo apt update
-      sudo apt install openjdk-8-jdk
-      ```
-    or
-      ```
-      sudo yum install java-1.8.0-openjdk
-      ```
-</details>
-
-<details>
-  <summary>Installing Docker...</summary>
-
-  Instructions for installing and configuring Docker can be found at
-  [https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/).  Windows users should follow the WSL2 instructions.
-  Docker Engine Version 18.03 (or newer) is needed to build.
-  Docker Engine Version 19.03 (or newer) is needed to build and run.
- 
-
-  Instructions for installing and configuring `docker-compose` can be found at
-  [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/).
-  Version 1.29 (or newer) is recommended; Version 1.27 (or newer) is required.
-</details>
-
-<details>
-  <summary>Docker RAM settings...</summary>
-
-  Tests run as part of the build process require at least 4GB of Docker RAM.  To check your Docker configuration, run:
-  ```
-  docker info | grep Memory
-  ```
-
-  By default, Docker on Mac is configured with 2 GB of RAM.  If you need to increase the memory on your Mac, click
-  on the Docker icon on the top bar and navigate to `Preferences->Resources->Memory`.
-
-  ![alt_text](docs/images/DockerConfigMac.png "Docker Configuration on a Mac")
-</details>
-
-<details>
-  <summary>If <code>docker run hello-world</code> does not work...</summary>
-
-  If `docker run hello-world` does not work, try the following:
-  1. [Is Docker running?](https://docs.docker.com/config/daemon/#check-whether-docker-is-running)
-      ```
-      docker info
-     ```
-  2. (Linux) [Are you in the `docker` user group?](https://docs.docker.com/engine/install/linux-postinstall/)
-      ```
-      sudo groupadd docker
-      sudo usermod -aG docker $USER
-      ```
-</details>
-
-
-### Checkout & Build Deephaven
-
-Once all of the required dependencies are installed and functioning, run:
+On Windows, these commands must be run using WSL 2.
+
+If any dependencies are missing or unsupported versions are installed, 
+see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
+
+
+### Create Deployment
+
+A directory must be created to store files and mount points for your deployment.
+Here, we are using the `deephaven-deployment` directory.  
+
+You will need to cd into the deployment directory to launch or interact with the deployment.
+
+```bash
+mkdir deephaven-deployment
+cd deephaven-deployment
+```
+
+### Launch: Python
```

Comment:
```suggestion
#### Launch: Python
```
Would it make sense to nest these launch commands underneath the `Create Deployment` header? We could also put the `These commands must be run from the deployment directory.` statement right before this header in order to not need to copy-paste that everywhere. No preference either way to me, just a thought I had

Reply:
I intentionally did not put them there.  Creating and launching are not the same.  Additionally, all of the following commands need to be run from the deployment director.  This includes things that are not just launch.

Reply:
Ah ok, that makes sense

Reply:
I'm not sure what to do with the `These commands must be run from the deployment directory.` line.  I think it is reasonable to put it in the `Create deployment` section, but I'm concerned that if it is only there, we will have users try to run the commands from other places without carefully reading the whole document.  We basically clean things up at the expense of potential user problems.

Reply:
I think I have an acceptable solution in.